doc/api/Renter.md

```diff
@@ -0,0 +1,277 @@
+Renter API
+==========
+
+This document contains detailed descriptions of the renter's API routes. For an
+overview of the renter's API routes, see [API.md#renter](/doc/API.md#renter).  For
+an overview of all API routes, see [API.md](/doc/API.md)
+
+There may be functional API calls which are not documented. These are not
+guaranteed to be supported beyond the current release, and should not be used
+in production.
+
+Overview
+--------
+
+The renter manages the user's files on the network. The renter's API endpoints
+expose methods for managing files on the network and managing the renter's
+allocated funds.
+
+Index
+-----
+
+| Route                                                         | HTTP verb |
+| ------------------------------------------------------------- | --------- |
+| [/renter](#renter-get)                                        | GET       |
+| [/renter](#renter-post)                                       | POST      |
+| [/renter/contracts](#rentercontracts-get)                     | GET       |
+| [/renter/downloads](#renterdownloads-get)                     | GET       |
+| [/renter/files](#renterfiles-get)                             | GET       |
+| [/renter/delete/___*siapath___](#renterdeletesiapath-post)    | POST      |
+| [/renter/download/___*siapath___](#renterdownloadsiapath-get) | GET       |
+| [/renter/rename/___*siapath___](#renterrenamesiapath-post)    | POST      |
+| [/renter/upload/___*siapath___](#renteruploadsiapath-post)    | POST      |
+
+#### /renter [GET]
+
+returns the current settings along with metrics on the renter's spending.
+
+###### JSON Response
+```javascript
+{
+  // Settings that control the behavior of the renter.
+  "settings": {
+    // Allowance dictates how much the renter is allowed to spend in a given
+    // period. Note that funds are spent on both storage and bandwidth.
+    "allowance": {  
+      // Amount of money allocated for contracts. Funds are spent on both
+      // storage and bandwidth.
+      "funds": "1234", // hastings
+
+      // Number of hosts that contracts will be formed with.
+      "hosts":24,
+
+      // Duration of contracts formed, in number of blocks.
+      "period": 6048, // blocks
+
+      // TODO: write a description for this field.
+      "renewwindow": 3024 // blocks
```

Comment:
Basically if there are less than 'renewwindow' blocks left in the contract, the contract is renewed